inz-demo/description.xml

```diff
@@ -1,18 +1,18 @@
 <SchemeManager version="7">
 	<Id>inz-demo</Id>
-	<Url>http://localhost:7000/</Url>
+	<Url>https://irma-scheme.local.internetnz.nz</Url>
 	<Demo>true</Demo>
 	<Name>
-		<en>INZ Demo</en>
-		<nl>INZ Demo</nl>
+		<en>INZ Scheme Manager Demo</en>
+		<nl>INZ Scheme Manager Demo</nl>
```

Comment:
I think we can remove Dutch language. in all files.

Reply:
At least the tag looks like it is required. It seems to me that it could at least be empty. I intend to do this cleanup at some point, but I want to do some few tests while changing language, that is why I left something on dutch.